src/components/Dashboard/Sidebar.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+const Sidebar = () => {
+  return (
+    <aside className="sidebar">
+      <ul className="sidebarList">
+        <li className="nav-link">
+          <NavLink to={"/"}>Home</NavLink>
+        </li>
+        <li className="nav-link">
+          <NavLink to={"/learning-objective"}>Learning Objective</NavLink>
+        </li>
+        <li className="nav-link">
+          <NavLink to={"/question-bank"}>Question bank</NavLink>
```

Comment:
No Need of brackets when passing a simple string to a component.

Reply:
done